css/obBase.css

```diff
@@ -202,13 +202,41 @@ form {
   width: calc(100% - 230px);
 }
 
-#pageNav {
+#appBar {
   position: fixed;
-  top: 0;
+  width: 100%;
+  z-index: 14;
+  background-color: #327EB8;
+  padding: 10px 8px;
+  box-sizing: border-box;
   -webkit-app-region: drag;
-  z-index: 13;
 }
 
+#appBar .windowControls {
+  position: absolute;
+  left: 8px;
+  top: 8px;  
+}
+
```

Comment:
You need 

#appBar .btn {
  -webkit-app-region: no-drag;
}

Otherwise the buttons aren't clickable and they don't have hover states.